src/components/cards/GenericCard.tsx

```diff
@@ -59,6 +60,7 @@ export const GenericCard = ({
         }
         borderRadius={20}
         shadow={color ? '18px accent' : '18px'}
+        style={{ flex: IS_IOS ? 0 : undefined }}
```

Comment:
on ios, `Box`s with undefined heights get messed up when a shadow is applied. for some reason this fixes that